third_party/terraform/tests/resource_compute_region_backend_service_test.go.erb

```diff
@@ -249,6 +249,13 @@ resource "google_compute_region_backend_service" "lipsum" {
 
   backend {
     group = "${google_compute_instance_group_manager.foobar.instance_group}"
+<% unless version.nil? || version == 'ga' -%>
+    failover = true
+  }
+
+  failover_policy {
+    failover_ratio = 0.4
```

Comment:
Maybe add the other added flags in here? Not sure if they can all go together, but may as well test as many of them here as possible